### PR TITLE
Implement LIC 13

### PR DESCRIPTION
**Pull request by Internet-Person-IP**
Implements LIC 13 with Tests Solves #8. Hopefully not too many mistakes.

### CODE REVIEW

File: src/main/java/CMV.java [hunk 1]
```diff
@@ -331,8 +332,23 @@ private boolean LIC12() {
...
+
+    int aPts = parameters.A_PTS;
+    int bPts = parameters.B_PTS;
+    double raduis1 = parameters.RADIUS1;
+    double raduis2 = parameters.RADIUS2;
+
+
+    for(int Pnt = 0; Pnt < points.length - aPts - bPts - 2; Pnt++){
+      if(insideCircle(points[Pnt], points[Pnt + aPts + 1], points[Pnt + aPts + bPts + 2], raduis1)) containedInCircleRad1 = true;
+      if(insideCircle(points[Pnt], points[Pnt + aPts + 1], points[Pnt + aPts + bPts + 2], raduis2)) containedInCircleRad2 = true;
```
Comment (K2017) [inline src/main/java/CMV.java:348]: Not sure if this checks with circles centered at the other points. Imagine a line of points and the test circle is only placed at the outer vertex of the line, then the other end may not be in the circle, but placed at the center the points would fit. Hope that made sense.

**Review by K2017 — CHANGES_REQUESTED** [review]
See comment

**Comment by Internet-Person-IP** [comment]
sorry, the commit messages are in past tense should I fix that?

**Comment by andreaskth** [comment]
Don't worry about it :) besides, I think it's probably a lot of hassle to change them after being pushed. I'll take a look at the code tonight or tomorrow and give my review 👍

File: src/test/java/LIC13Test.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+  }
+
+  @Test
+  /*
+  * It tests that LIC13 is True when the correct conditions are met
+  */
+  public void testLIC13True(){
+
+    Parameters params = new Parameters();
```
Comment (andreaskth) [inline src/test/java/LIC13Test.java:38]: Since there was a minor error in the LIC13-function (looking at <= RADIUS1 instead of > RADIUS1), this test will probably fail when that is fixed. Or at least I can't see three points that couldn't be encompassed by a circle of radius 1?

File: src/test/java/LIC13Test.java [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+  
+  
+  @Test
+  /*
+  * It tests a Edge case
+  * Tests if LIC14 can calculate AREAS correctly close to the end the Array.
+  */
+  public void testLIC14EdgeCase(){
+
+    Parameters params = new Parameters();
```
Comment (andreaskth) [inline src/test/java/LIC13Test.java:81]: Should this method be in another file, or did you forget to rename from 14 to 13?

**Review by andreaskth — CHANGES_REQUESTED** [review]
Looks pretty good, some minor comments. Shouldn't take too long to fix though 🚀

File: src/main/java/CMV.java [hunk 4]
```diff
@@ -426,8 +426,22 @@ private boolean LIC12() {
...
+
+    int aPts = parameters.A_PTS;
+    int bPts = parameters.B_PTS;
+    double raduis1 = parameters.RADIUS1;
+    double raduis2 = parameters.RADIUS2;
+
+    for(int Pnt = 0; Pnt < points.length - aPts - bPts - 2; Pnt++){
+      if(Point.smallestCircle(points[Pnt], points[Pnt + aPts + 1], points[Pnt + aPts + bPts + 2]) <= raduis1) containedInCircleRad1 = true;
+      if(Point.smallestCircle(points[Pnt], points[Pnt + aPts + 1], points[Pnt + aPts + bPts + 2]) <= raduis2) containedInCircleRad2 = true;
+    }
```
[the remark this answers is not in the document]
Comment (Internet-Person-IP) [reply src/main/java/CMV.java:442]: oh sorry, I can't read. Gotta Fix That

File: src/test/java/LIC13Test.java (hunk 3 above)
Comment (Internet-Person-IP) [reply src/test/java/LIC13Test.java:81]: Forgot to rename

**Review by andreaskth — APPROVED** [review]
All seems good now, nice job!

**Review by K2017 — APPROVED** [review]
LGTM
